terraform/main.tf

```diff
@@ -31,4 +31,122 @@ resource "aws_s3_bucket_lifecycle_configuration" "bhattis-coughsense_expiration"
 	  days = 100
     }
   }
+}
+
+resource "aws_iam_role" "ecs_task_execution_role" {
+  name = "ecs_task_execution_role_v1"
```

Comment:
anything with a `name` field i would prefix you project name to it so it doesn't overlap with others' resources in AWS